include/state-observation/observer/tilt-estimator-humanoid.hpp

```diff
@@ -0,0 +1,159 @@
+/**
+ * \file      tilt-estimator.hpp
+ * \author    Rafael Cisneros, Mehdi Benallegue
+ * \date       2018
+ * \brief      Defines the class for the tilt estimator.
+ *
+ * \details
+ *
+ *
+ */
+
+#ifndef TILTESTIMATORHUMANOIDHPP
+#define TILTESTIMATORHUMANOIDHPP
+
+#include <state-observation/api.h>
+#include <state-observation/observer/tilt-estimator.hpp>
+#include <state-observation/observer/zero-delay-observer.hpp>
+
+namespace stateObservation
+{
+
+/**
+ * \class  TiltEstimatorHumanoid
+ * \brief
+ *         Description is pending
+ *
+ *         use getEstimatedState to obtain the state vector
+ *         the tilt R.transpose()*e_z is constituted
+ *         with the last three components of the state vector.
+ *
+ */
+class STATE_OBSERVATION_DLLAPI TiltEstimatorHumanoid : public TiltEstimator
```

Comment:
That's the same description as `TiltEstimator`. What is the purpose of this separate class?

Reply:
TiltEstimatorHumanoid implements functions that are necessary only for a use with humanoid robots so we want to keep a lightweight version of TiltEstimator that does not require them

Reply:
Can you update the documentation to reflect that?